src/internal/views/nunjucks/billing/batch-list.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "badge.njk" import badge %}
+{% from "paginate.njk" import paginate %}
+{% block content %}
+
+{{ title(pageTitle) }}
+{{ formRender(form) }}        
```

Comment:
If this form is simply a button that leads to the `create bill run flow`, it can be implemented with a link styled as a button.
See the 'element' macro option here:
https://design-system.service.gov.uk/components/button/

Reply:
done - thanks, I wasn't sure about this.